var/spack/repos/builtin/packages/py-bbp-workflow/package.py

```diff
@@ -12,28 +12,28 @@ class PyBbpWorkflow(PythonPackage):
     homepage = 'https://bbpcode.epfl.ch/code/#/admin/projects/nse/bbp-workflow'
     git      = 'ssh://bbpcode.epfl.ch/nse/bbp-workflow'
 
-    version('2.1.8', tag='bbp-workflow-v2.1.8')
+    version('2.1.11', tag='bbp-workflow-v2.1.11')
 
     depends_on('py-setuptools', type=('build'))
 
-    depends_on('py-luigi', type='run')
-    depends_on('py-luigi-tools', type='run')
-    depends_on('py-entity-management', type='run')
+    depends_on('py-luigi',               type='run')
+    depends_on('py-luigi-tools',         type='run')
+    depends_on('py-entity-management',   type='run')
     depends_on('py-requests-unixsocket', type='run')
-    depends_on('py-sh', type='run')
-    depends_on('py-matplotlib', type='run')
-    depends_on('py-dask+diagnostics', type='run')
-    depends_on('py-distributed', type='run')
-    depends_on('py-xarray', type='run')
-    depends_on('py-zarr', type='run')
-    depends_on('py-notebook', type='run')
-    depends_on('py-ipyparallel', type='run')
+    depends_on('py-sh',                  type='run')
+    depends_on('py-matplotlib',          type='run')
+    depends_on('py-dask+diagnostics',    type='run')
+    depends_on('py-distributed',         type='run')
+    depends_on('py-xarray',              type='run')
+    depends_on('py-zarr',                type='run')
+    depends_on('py-notebook',            type='run')
+    depends_on('py-ipyparallel',         type='run')
```

Comment:
Minor, I prefer to avoid this type of alignment because you'll need to fix all the lines if you add other packages with longer names.
However, I'm fine anyway because I see that it's already used in other spack packages as well (and `black` is not used).

Reply:
Cool, I look forward to it 😄

Reply:
:facepalm: :smile: